tox.ini

```diff
@@ -63,6 +64,7 @@ deps =
     py27: subprocess32==3.2.7
     pypy: psycopg2cffi-compat==1.1
     py{27,35,36,37}: psycopg2-binary==2.7.4
+    py38: psycopg2-binary
```

Comment:
```suggestion
    py{27,35,36,37,38}: psycopg2-binary==2.7.4
```

does it work?

Reply:
No of course it doesnt work.  That is why I added this.

Reply:
thanks for pinning